TabBarFlags.go

```diff
@@ -0,0 +1,30 @@
+package imgui
+
+const (
+
+	// TabBarFlagsNone default = 0.
+	TabBarFlagsNone = 0
+	// TabBarFlagsReorderable Allow manually dragging tabs to re-order them + New
+	// tabs are appended at the end of list
+	TabBarFlagsReorderable = 1 << 0
+	// TabBarFlagsAutoSelectNewTabs Automatically select new tabs when they appear
+	TabBarFlagsAutoSelectNewTabs = 1 << 1
+	// TabBarFlagsNoCloseWithMiddleMouseButton Disable behavior of closing tabs (that are submitted with p_open != NULL) with middle mouse button. You can
+	// still repro this behavior on user's side with if (IsItemHovered() && IsMouseClicked(2)) *p_open = false.
+	TabBarFlagsNoCloseWithMiddleMouseButton = 1 << 2
+	// TabBarFlagsNoTabListPopupButton Undocumented in Dear imGui v1.67
```

Comment:
Although it is undocumented, it probably has a function.
Please, for this, and similar that don't have any information in the .h file, add information what they would do.